src/common/latest/config.ts

```diff
@@ -1,6 +1,6 @@
 // When the user does not specify a specific version, these willl be the default versions used.
 const msBuildVersion = "6.2.0.85879";
-const cliVersion = "6.0.0.4432";
+const cliVersion = "6.1.0.4477";
```

Comment:
do these versions still align?

Reply:
They don't have anymore thanks to [this commit](https://github.com/SonarSource/sonar-scanner-azdo/pull/388/commits/a3c02628133ad6ce2d2418b20fc786416193e228) 😎